oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/indicator/WithMetadata.java

```diff
@@ -16,13 +16,13 @@
  *
  */
 
-package org.apache.skywalking.oap.server.core.alarm;
+package org.apache.skywalking.oap.server.core.analysis.indicator;
 
 /**
- * Alarm supported interface implementor could return the {@link AlarmMeta}
+ * Alarm supported interface implementor could return the {@link IndicatorMetaInfo}
```

Comment:
From the action of class name change, I think this class will not just for support alarm.

Reply:
Fixed.